.github/workflows/ci.yml

```diff
@@ -264,95 +270,3 @@ jobs:
         with:
           command: build
           args: --verbose --release
-
-  clippy-deps:
-    name: Clippy (stable)
-    timeout-minutes: 30
-    runs-on: ubuntu-latest
-    env:
-      CARGO_INCREMENTAL: 0
-      RUST_BACKTRACE: full
-
-    steps:
-      - uses: actions/checkout@v2.4.0
-        with:
-          persist-credentials: false
-
-      - uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable
-          components: clippy
-          override: true
-
-      - uses: Swatinem/rust-cache@v1
-
-      - name: Show env vars
-        run: |
-            echo "Common env vars:"
-            echo "RUST_BACKTRACE=${{ env.RUST_BACKTRACE }}"
-            echo "Build env vars:"
-            echo "CARGO_INCREMENTAL=${{ env.CARGO_INCREMENTAL }}"
-            echo "CARGO_TARGET_DIR=${{ env.CARGO_TARGET_DIR }}"
-
-      - name: Run clippy
-        uses: actions-rs/clippy-check@v1.0.7
-        with:
-          # GitHub displays the clippy job and its results as separate entries
-          name: Clippy (stable) Results
-          token: ${{ secrets.GITHUB_TOKEN }}
-          args: --all-features --all-targets -- -D warnings
-
-      # This check makes sure the crate dependency check is accurate
-      - name: Check Cargo.lock is up to date
-        uses: actions-rs/cargo@v1.0.3
-        with:
-          command: check
-          args: --locked --all-features --all-targets
-
-      # Edit zebra/deny.toml to allow duplicate dependencies
-      - name: Check for dependent crates with different versions
-        uses: EmbarkStudios/cargo-deny-action@v1.2.9
-        with:
-          command: check bans
-          args: --all-features --workspace
-
-      - name: Check crate sources
-        uses: EmbarkStudios/cargo-deny-action@v1.2.9
-        with:
-          command: check sources
-          args: --all-features --workspace
```

Comment:
Blocker: it looks like these 3 checks were deleted.

Reply:
Solved witn https://github.com/ZcashFoundation/zebra/pull/3377/commits/11e24d94f8090c0fca9644d79412f58a8a654f92